### ISSUE TITLE
Project contracts disappear when moving between contracts in linked projects

### ISSUE THREAD

**Issue by sisyphusSmiling** [onflow/flow-playground#662]
**Describe the bug**

When opening a playground project, contracts disappear if not deployed before switching between contract views.

**To Reproduce**

1. Go to this FungibleTokens tutorial project: https://play.onflow.org/e63bfce9-3324-4385-9542-626845ae0363
1. Before deploying any contracts, select `Contract 2` from the sidebar.
1. Select `Contract 1` and see that the contract has disappeared.
1. Now exit the playground tab and go back to the project linked above
1. This time, deploy the `BasicToken` contract (AKA `Contract 1`)
1. Repeat 3 & 4 and notice that the contract stays the same

**Expected behavior**

I would expect that clicking on a project link would take me to the loaded project and that I could view other contracts before deploying without the project's contracts disappearing.

**Desktop (please complete the following information):**

 - OS: MacOS 13.3.1 (a) 
 - Browser: Chrome
 - Version: 112.0.5615.137 (Official Build) (arm64)

**Comment by bthaile** [comment]
Thanks for bring this to our attention. I see the issue, This will be an issue with the tutorials or shared projects. 
Looking at it, it appears the urls to the files are not updated from local-project.

**Comment by bthaile** [comment]
@sisyphusSmiling  can you verify that this issue has been fixed? Thanks

**Comment by sisyphusSmiling** [comment]
Looks like it's working @bthaile, thanks!

**Comment by joshuahannan** [comment]
@bthaile It looks like the original issue was fixed, but I just discovered that it happens again when clicking on an account and going back to a contract before deploying.

To reproduce:

* Open this link (from the tutorial): https://play.onflow.org/f08e8e0d-d28e-4cbe-8d72-3afe2349c629
* See that `ExampleNFT` is in contract 1
* Click on account 1
* Click one contract 1 again and it has been replaced with hello world again.

Is there a more thorough test we can do once this is fixed to make sure it isn't happening in any of the cases?
